model-optimizer/extensions/middle/reverse_tensor_iterator.py

```diff
@@ -18,6 +19,7 @@ class ReverseTensorIteratorLSTM(MiddleReplacementPattern):
     """
 
     enabled = True
+    force_clean_up = True
```

Comment:
why do we need it?

Reply:
Becase seq_length can be constant(if ReverseSequence created not by Reverse2ReverseSequence transformation) or subgraph with shapeOf and broadcast and it should be deleted from graph (we do only disconnect)